.buildkite/commands/run-ui-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+TEST_NAME=$1
+DEVICE=$2
+IOS_VERSION=$3
+
+echo "Running $TEST_NAME on $DEVICE for iOS $IOS_VERSION"
```

Comment:
Worth adding `set -u` or combining with the already present `set -e` using `set -eu` so that the script will fail if these three values are not set.

I learned about `set -u` just recently, BTW. `u` stands for unbound, as in "fail on unbound variables".

I checked this locally with a dedicated script:

```sh
#!/bin/bash

set -eu

TEST_NAME=$1
DEVICE=$2
IOS_VERSION=$3

echo "Running $TEST_NAME on $DEVICE for iOS $IOS_VERSION"
```

```
$ sh ./t.sh
./t.sh: line 5: $1: unbound variable

$ sh ./t.sh a b c
Running a on b for iOS c
```

Reply:
I ended up force-pushing a bunch to clean this up, but this is addressed in 31495d7242f7be8cde610a89b611b65eb070a257